cmd/ebrelayer/replay.go

```diff
@@ -168,7 +169,7 @@ func RunReplayCosmosCmd(cmd *cobra.Command, args []string) error {
 }
 
 // RunListMissedCosmosEventCmd executes initRelayerCmd
-func RunListMissedCosmosEventCmd(cmd *cobra.Command, args []string) error {
+func RunListMissedCosmosEventCmd(_ *cobra.Command, args []string) error {
 	// Parse flag --rpc-url
 	rpcURL := viper.GetString(FlagRPCURL)
```

Comment:
Replace viper usage and standardise rpc / node flag / arg

Reply:
Removed viper, still need to standardise rpc / node flag / arg

Reply:
As far as I can tell, the right thing to use is the node argument, so I just removed rpc-url.